util/metrics/metrics.go

```diff
@@ -91,4 +91,20 @@ var (
 	TransactionMessagesDroppedFromBacklog = MetricName{Name: "algod_transaction_messages_dropped_backlog", Description: "Number of transaction messages dropped from backlog"}
 	// TransactionMessagesDroppedFromPool "Number of transaction messages dropped from pool"
 	TransactionMessagesDroppedFromPool = MetricName{Name: "algod_transaction_messages_dropped_pool", Description: "Number of transaction messages dropped from pool"}
+	// TransactionMessagesAlreadyCommitted "Number of duplicate or error transaction messages before placing into a backlog"
+	TransactionMessagesAlreadyCommitted = MetricName{Name: "algod_transaction_messages_already_committed", Description: "Number of duplicate or error transaction messages after txhandler backlog"}
```

Comment:
Maybe rename the tag name to be consistent with the description? 
```suggestion
	TransactionMessagesAlreadyCommitted = MetricName{Name: "algod_transaction_messages_err_or_already_committed", Description: "Number of duplicate or error transaction messages after txhandler backlog"}
```

Reply:
I prefer to keep the name shorted, the description mentions errors